### PR TITLE
Add itemClass & linkClass customizations

### PR DESCRIPTION
**Pull request by ened**
Some themes might need extra markup for their paginator components.

### CODE REVIEW

**Comment by ened** [comment]
Updated to pass validation suite `npm run validate`. @wagoid  interested?

File: src/components/Page.js [hunk 1]
```diff
@@ -37,19 +41,26 @@ export default class Page extends Component {
...
           disabledClass,
           isActive,
           isDisabled,
         } = this.props;
 
         const css = cx({
           [activeClass]: isActive,
-          [disabledClass]: isDisabled
+          [disabledClass]: isDisabled,
+          [itemClass]: itemClass !== undefined,
```
Comment (wagoid) [inline src/components/Page.js]: `classnames` already ignores falsy values, so you can just pass `itemClass` like this:
```javascript
const css = cx(itemClass, {
  [activeClass]: isActive,
  [disabledClass]: isDisabled
});
```

You can see further information about this here: https://github.com/JedWatson/classnames#usage

File: src/components/Page.js [hunk 2]
```diff
@@ -37,19 +41,26 @@ export default class Page extends Component {
...
+        });
+				
+        const linkCss = cx({
```
Comment (wagoid) [inline src/components/Page.js]: Same here 🙂

**Comment by ened** [comment]
@wagoid fixed. Btw thx for sharing about `classnames` .. happy to learn something new..

**Pull request merged**